src/main/java/seedu/address/model/person/insurance/InsurancePlansManager.java

```diff
@@ -105,6 +112,25 @@ public void checkIfPlanNotOwned(InsurancePlan plan) throws ParseException {
         }
     }
 
+    /**
+     * Adds a claim to the insurance plan of the client.
+     *
+     * @param insurancePlan The insurance plan the claim is to be added to.
+     * @param claim         The claim that is to be added to the insurance plan.
+     * @throws CommandException if the claimId already exists.
+     */
+    public void addClaimToInsurancePlan(InsurancePlan insurancePlan, Claim claim) throws CommandException {
+        if (this.claimIds.contains(claim.getClaimId())) {
+            throw new CommandException(DUPLICATE_CLAIM_ID_MESSAGE);
+        }
+        for (InsurancePlan p : insurancePlans) {
```

Comment:
hmmm, not too sure about the use of a single letter variable here (but happy to be corrected). We could use `insurancePlan` in place of p

Reply:
I think keeping as p is okay too since this part of the code is only 3 lines

Reply:
I agree with Zi Yi, using insurancePlan will make it too cluttered.